pkg/logging/filelogging.go

```diff
@@ -0,0 +1,56 @@
+package logging
```

Comment:
this package `logging` is incorrect abstraction. I would rather see this as function in assets package that returns a string for a list of Assets provided to it as summary.